tests/search/containers/TestCurrentSearchPage.js

```diff
@@ -31,19 +31,35 @@ describe('CurrentSearchPage.mapDispatchToProps', () => {
     Object.keys(mocks).forEach((name) => mocks[name].restore());
   });
 
-  function handleSearch(query, response = Promise.resolve({})) {
-    mocks.api.expects('search').withArgs({ query }).once().returns(response);
-    return mapDispatchToProps(dispatch).handleSearch(query);
+  function handleSearch(query, page, response = Promise.resolve({}), expectedPage) {
+    mocks.api.expects('search')
+      .withArgs({ page: expectedPage || page, query })
+      .once()
+      .returns(response);
+    return mapDispatchToProps(dispatch).handleSearch(query, page);
   }
 
   it('sets the query', () => {
     const searchAction = sinon.stub();
     mocks.actions
       .expects('searchStart')
-      .withArgs('DuckDuckGo')
+      .withArgs('DuckDuckGo', 5)
+      .once()
+      .returns(searchAction);
+    return handleSearch('DuckDuckGo', 5).then(() => {
+      assert(dispatch.calledWith(searchAction), 'expected action to be dispatched');
+      mocks.actions.verify();
+    });
+  });
+
+  it('sets defaults the page to 1', () => {
+    const searchAction = sinon.stub();
+    mocks.actions
+      .expects('searchStart')
+      .withArgs('DuckDuckGo', 1)
       .once()
       .returns(searchAction);
-    return handleSearch('DuckDuckGo').then(() => {
+    return handleSearch('DuckDuckGo', undefined, undefined, 1).then(() => {
```

Comment:
Seeing this makes me think an object for optional args would help make it clearer what's what.